pullRequestReview.js

```diff
@@ -0,0 +1,70 @@
+var express = require('express');
+
+import { request, abort, } from 'request-promise';
+
+const port = process.env.PORT || 1337;
+
+const app = express();
+
+async function incoming2outgoingPayload(payload) {
+  import jsontoxml from 'jsontoxml';
+  const itemsToXML = {}
+  for(let i = 0; i <= payload.items.length; i++) {
+    const startDate = new Date(payload.items[i].start);
+    const endDate = new Date(payload.items[i].end);
+    itemsToXML[`vacation-${startDate.toString()}-${endDate.toString()}`] = payload.items[i].approved == true ? true : false;
+  }
```

Comment:
better to use `reduce` here because it conveys the intent of what the function is doing a bit better. A `for` loop could be doing anything, but in this case a `reduce` would show that you are accumulating an array of data into a single object.

If you need help with the actual reduce function lmk, this is a pretty straightforward use case